src/server/editorServices.ts

```diff
@@ -189,9 +189,9 @@ import {
 } from "./_namespaces/ts.server.js";
 import * as protocol from "./protocol.js";
 
-export const maxProgramSizeForNonTsFiles = 20 * 1024 * 1024;
+export const maxProgramSizeForNonTsFiles: number = 20 * 1024 * 1024;

```

Comment:
Is this a bug in isolatedDeclarations? Why is this not inferred?

Reply:
I still sort of feel like this is an oops in ID, like, we can const evaluate left shifts, why not multiplication?

Reply:
Yup, I'm filing an issue on this

Reply:
left and right shifts can't overflow `Number.MAX_SAFE_INTEGER`